### PR TITLE
HSD8-73 Linked event speaker to person node if available

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Add link to person node from speaker mapping.

# Needed By (Date)
- ASAP

# Urgency
- Low

# Steps to Test

1. Checkout this branch
1. `composer update --prefer-source`
1. `blt drupal:sync --site=archaeology --sync-files`
1. create a person node with the name "Thomas Schultz"
1. Change the Event Image importer url: `drush @archaeology.local cset migrate_plus.migration.hs_events_image_importer source.urls "https://events.stanford.edu/xml/drupal/v2.php?organization=19"`
1. Change the Event Node importer url: `drush @archaeology.local cset migrate_plus.migration.hs_events_importer source.urls "https://events.stanford.edu/xml/drupal/v2.php?organization=19"`
1. run the imports `drush @archaeology.local cron`
1. find the event named "THOMAS SCHULTZ SUMMER PIANO SEMINAR: Piano Masterclass with Thomas Schultz"
1. Review the speaker field
1. validate the speaker person is set to the person node you created.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: config/archaeology/migrate_plus.migration.hs_events_importer.yml [hunk 1]
```diff
@@ -185,14 +185,24 @@ process:
...
+    value_key: title
+    bundle_key: type
+    bundle: hs_person
+    entity_type: node
+    ignore_case: true
   field_hs_event_speaker:
     -
       source: speaker
-      plugin: entity_generate_no_lookup
+      plugin: hs_entity_generate
```
Comment (sherakama) [inline config/archaeology/migrate_plus.migration.hs_events_importer.yml]: Why the override? Can you leave a code comment here?

Comment (pookmish) [reply config/archaeology/migrate_plus.migration.hs_events_importer.yml]: I think a decent comment explanation is in the plugin class. https://github.com/SU-HSDO/suhumsci/pull/72/files#diff-acab96968cfb59d1284c3f126cc3ad23R23

**Comment by boznik** [comment]
this is working as expected.

**Comment by boznik** [comment]
this is cool by me @sherakama

**Pull request merged**